### PR TITLE
#2374 Fix typos in various places

### LINKED ISSUE #2374: Fix minor typos
### Problem

Found minor typos while testing. 

"Failed to login" => "Failed to log in"
- https://github.com/okTurtles/group-income/blob/9eb4aba0587475feebe209d42b141ecfedca07ee/frontend/controller/app/identity.js#L348
- https://github.com/okTurtles/group-income/blob/9eb4aba0587475feebe209d42b141ecfedca07ee/frontend/controller/actions/identity.js#L322

IFF there's only one: "members" => "member"
- https://github.com/okTurtles/group-income/blob/9eb4aba0587475feebe209d42b141ecfedca07ee/frontend/views/pages/Payments.vue#L107
- (possibly) https://github.com/okTurtles/group-income/blob/9eb4aba0587475feebe209d42b141ecfedca07ee/frontend/views/containers/contributions/ContributionsWidget.vue#L139
- others?

### Solution

Fix

### PR DESCRIPTION
**Pull request by SebinSong**
closes #2374 

[ Some screenshots for the fixes ]

copy-fix for payment table
<br><br>

### CODE REVIEW

**Comment by SebinSong** [comment]
@taoeffect @dotmacro 

I found another place for a typo so made a fix for that too:

File: frontend/views/pages/GroupChat.vue [hunk 1]
```diff
@@ -75,12 +75,14 @@ page(pageTestName='groupChat' :miniHeader='isGroupDirectMessage()')
...
-        i18n.is-unstyled.c-link(
-          tag='button'
+        button.is-unstyled.c-link(
+          type='button'
           @click='openModal("ChatMembersAllModal")'
-          :args='{ numMembers: summary.numberOfMembers  }'
           data-test='channelMembers'
-        ) {numMembers} members
+        )
+          i18n(v-if='summary.numberOfMembers === 1' :args='{ num: summary.numberOfMembers }') {num} member
```
Comment (taoeffect) [inline frontend/views/pages/GroupChat.vue]: In this case, there's no need for the `{num}` variable in the string or the `:args` :), you know it's `1`

File: frontend/views/pages/Payments.vue [hunk 2]
```diff
@@ -421,10 +416,13 @@ export default ({
...
-      return {
-        amount: this.withGroupCurrency(amount),
-        count: this.paymentsTodo.length
-      }
+      const memberIds = uniq(this.paymentsTodo.map(item => item.toMemberID))
+      const membersLen = memberIds.length
+      const args = { amt: this.withGroupCurrency(amount), count: membersLen }
+
+      return membersLen === 1
+        ? L('{amt} in total, to {count} member', args)
```
Comment (taoeffect) [inline frontend/views/pages/Payments.vue]: Same here, can remove `{count}`

File: frontend/views/containers/contributions/ContributionsWidget.vue [hunk 3]
```diff
@@ -154,9 +154,14 @@ export default ({
...
+          const isSingularCount = count === 1
           status = givingNonMonetary
-            ? L('You and {count} other members are contributing.', { count })
-            : L('{count} members are contributing.', { count })
+            ? isSingularCount
+              ? L('You and {count} other member are contributing.', { count })
+              : L('You and {count} other members are contributing.', { count })
+            : isSingularCount
+              ? L('{count} member is contributing.', { count })
+              : L('{count} members are contributing.', { count })
```
Comment (taoeffect) [inline frontend/views/containers/contributions/ContributionsWidget.vue]: Here too

**Review by taoeffect — CHANGES_REQUESTED** [review]
Nice work, and great job noticing other areas! I'm requesting that we don't include the variable for singular as it will hopefully clear things up a bit for the translators

File: frontend/views/pages/GroupChat.vue (hunk 1 above)
Comment (SebinSong) [reply frontend/views/pages/GroupChat.vue]: Right, it's a good point.

**Review by taoeffect — APPROVED** [review]
Nice!

**Pull request merged**